lib/rouge/lexers/squirrel.rb

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*- #
+
```

Comment:
Please add '# frozen_string_literal: true'

Reply:
Just added it. Sorry for the delay.